node/packages/parser/src/ethereum/__tests__/mockData/mempoolMock.ts

```diff
@@ -0,0 +1,15 @@
+import { Tx } from '@shapeshiftoss/blockbook'
+
+export const mempoolMock = (tx: Tx, tokenTransfers = false) => {
+  const ethereumSpecificOverride = { status: -1, gasUsed: null }
+  const mempoolSpecific = {
+    blockHeight: -1,
+    confirmations: 0,
+    fees: '0',
+    blockHash: undefined,
+    rbf: true,
```

Comment:
This would technically be a dynamic value determined by how the transaction was broadcast and not always true for every mempool tx case. Another value we don't currently look at however.

Reply:
Ah I see - cool if I remove this from the data for now, then?

Reply:
That works with me.

Reply:
Actioned in https://github.com/shapeshift/unchained/pull/283/commits/eb15ccb4880aaf1be529a5cc883f9b2151f44d97